app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<div class="category-header">
+     <a href="<%= categories_path %>">
+            <i class="fa-solid fa-arrow-left"></i>
+        </a>
+    <h2>New Category</h2>
+    <a href="/logout">
+        <i class="fa-solid fa-right-from-bracket"></i>
+    </a>
+</div>
+<%= form_with(model: @category)do |form| %>
+    <div class="form-container">
+        <%= form.text_field :name, placeholder: "Name" %>
+        <%= form.text_field :icon, placeholder: "Icon Link" %>
+         <%= form.submit "Create", class: "submit-button" %>
+    </div>
```

Comment:
- [ ] Kindly fix the submit button for the categories, the requirement is that when a user clicks the submit button, the app navigates to the home page.

![selection](https://github.com/atok624/Budget-App/assets/53124548/b3ffac4a-9199-4306-a01a-081cd4c11d65)

Reply:
Resolve